### ISSUE TITLE
issue about latest rev. 3485 [DB/Creature]: Greatly reduce respawn time for a few specific named mobs in low level areas

### ISSUE THREAD

**Issue by duzyizly** [Atlantiss/NetherwingBugtracker#7212]
@Wolffenstein 
Just a note...
Before rev. 3485 those NPCs used to spawn after 40-45 seconds.
Now they are spawning after exact 60 sec.
So... Not so **"reduced respawn time"** : )

Tested with NPCs in tirisfal glades (agamand hills) and Zalzane (echo isles) - so I belive timer is set to all "quest mini-bosses".

Not sure if intended.

**Comment by Easyfraggy** [comment]
The change was only to level 1-6 named mobs, 60 seconds is indeed too much on Zalazane etc. Can you gather a list of all relevant NPC IDs of mobs outside the 1-6 zones?

The ones that were changed in recent update were: 
3281
3183
8554
15367
1919
1667
808
103
16522

All of them in the starting areas lvl 1-6.

**Comment by duzyizly** [comment]
I can help only with **Horde 6-12 zones** since I'm not familiar with alliance starting zones.

1. Tirisfal Glades:
1657
1654
1655
1656
1753
1662
1665
1664

2. Durotar:
3203
3205
3204
3192

3. Eversong Woods
8479
15408
15409
16294
15949

4. NPC i'm not sure about (requires pre-quest to make him aggresive)
15420

Tested **all** of them in last 45 min. They tend to respawn after 60-63 seconds (longest [unique] respawn I noticed was 67 sec NPC=3192)
